x-pack/platform/plugins/private/painless_lab/public/styles/_index.scss

```diff
@@ -1,47 +0,0 @@
-@import '../../../../../../../src/core/public/mixins';
-
-/**
- * This is a very brittle way of preventing the editor and other content from disappearing
- * behind the bottom bar.
- */
-$bottomBarHeight: $euiSize * 3;
-
-.painlessLabBottomBarPlaceholder {
-  height: $bottomBarHeight;
-}
-
-.painlessLabLeftPane {
-  padding-top: $euiSizeM;
-  background-color: $euiColorEmptyShade;
-}
-
-.painlessLabRightPane {
-  background-color: $euiColorEmptyShade;
-  padding: $euiSizeS;
-  border-left: $euiBorderThin;
-  height: 100%;
-}
-
-.painlessLabRightPane__tabs {
-  display: flex;
-  flex-direction: column;
-  height: 100%;
-
-  [role='tabpanel'] {
-    height: 100%;
-    overflow-y: auto;
-  }
-}
-
-// adding dev tool top bar + bottom bar height to the body offset
-// (they're both the same height, hence the x2)
-$bodyOffset: $bottomBarHeight * 2;
-
-.painlessLabMainContainer {
-  @include kibanaFullBodyHeight($bodyOffset);
-}
```

Comment:
I could have removed only this one... but I got carried over and ported all the CSS in this file 😬